### PR TITLE
Update section per legal advisory

### PR DESCRIPTION
**Pull request by rgreenberg1**
also will need to update the new name of the NM License, soon

### CODE REVIEW

File: src/content/products/deepsparse/community.mdx [hunk 1]
```diff
@@ -258,8 +258,8 @@ For more general questions about Neural Magic, complete this [form.](http://neur
 
 ### License
 
-The Community Edition of the project's binary containing the DeepSparse Engine is licensed under the [Neural Magic Engine License.](https://github.com/neuralmagic/deepsparse/blob/main/LICENSE-NEURALMAGIC)
-Example files and scripts included in this repository are licensed under the [Apache License Version 2.0](https://github.com/neuralmagic/deepsparse/blob/main/LICENSE) as noted.
+The Community Edition of the project's binary containing DeepSparse is licensed under the [Neural Magic Engine License.](https://github.com/neuralmagic/deepsparse/blob/main/LICENSE-NEURALMAGIC)
+Some source code, example files, and scripts included in the GitHub repository or directory are licensed under the [Apache License Version 2.0](https://github.com/neuralmagic/deepsparse/blob/main/LICENSE) as noted.
```
Comment (jeanniefinks) [inline src/content/products/deepsparse/community.mdx:262]:
```suggestion
Some source code, example files, and scripts included in the deepsparse GitHub repository or directory are licensed under the [Apache License Version 2.0](https://github.com/neuralmagic/deepsparse/blob/main/LICENSE) as noted.
```

File: src/content/products/deepsparse/community.mdx [hunk 2]
```diff
@@ -258,8 +258,8 @@ For more general questions about Neural Magic, complete this [form.](http://neur
 
 ### License
 
-The Community Edition of the project's binary containing the DeepSparse Engine is licensed under the [Neural Magic Engine License.](https://github.com/neuralmagic/deepsparse/blob/main/LICENSE-NEURALMAGIC)
-Example files and scripts included in this repository are licensed under the [Apache License Version 2.0](https://github.com/neuralmagic/deepsparse/blob/main/LICENSE) as noted.
+The Community Edition of the project's binary containing DeepSparse is licensed under the [Neural Magic Engine License.](https://github.com/neuralmagic/deepsparse/blob/main/LICENSE-NEURALMAGIC)
```
Comment (jeanniefinks) [inline src/content/products/deepsparse/community.mdx:261]:
```suggestion
DeepSparse Community is licensed under the [Neural Magic DeepSparse Community License.](https://github.com/neuralmagic/deepsparse/blob/main/LICENSE-NEURALMAGIC)
```

File: src/content/products/deepsparse/community.mdx [hunk 3]
```diff
@@ -258,8 +258,8 @@ For more general questions about Neural Magic, complete this [form.](http://neur
...
+Some source code, example files, and scripts included in the GitHub repository or directory are licensed under the [Apache License Version 2.0](https://github.com/neuralmagic/deepsparse/blob/main/LICENSE) as noted.
 
 [The Enterprise Edition](https://docs.neuralmagic.com/products/deepsparse-ent) requires a Trial License or [can be fully licensed](https://neuralmagic.com/legal/master-software-license-and-service-agreement/) for production, commercial applications.
```
Comment (jeanniefinks) [inline src/content/products/deepsparse/community.mdx:264]:
```suggestion
[DeepSparse Enterprise](https://docs.neuralmagic.com/products/deepsparse-ent) requires a Trial License or [can be fully licensed](https://neuralmagic.com/legal/master-software-license-and-service-agreement/) for production, commercial applications.
```

**Review by jeanniefinks — CHANGES_REQUESTED** [review]
minor edits

**Review by jeanniefinks — APPROVED** [review]

**Review by Beth-Kosis — APPROVED** [review]
These changes look fine.

**Pull request merged**